test/functionalTest/cases/1444_err_msg_duplicate_attr/create_entity_dup_attr.test

```diff
@@ -0,0 +1,71 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entity V2 To Many Entities
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, creating first entity with id E1
+# 02. POST /v2/entities, creating second entity with id E1
+# 03. POST /v2/entities, creating third entity with id E1
+# 04. POST /v2/entities, creating fourth entity with id E1
+# 05. POST /v2/entities, creating fifth entity with id E1
+# 06. GET /v2/entities/E1 (too many entities error)
+#
+
+echo "01. POST /v2/entities, creating first entity with id E1"
+echo "=========================================================="
+payload='{
+    "id": "E1",
+    "type": "T1",
+    "attr": "value",
+    "attr": 1
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+
+--REGEXPECT--
+01. POST /v2/entities, creating first entity with id E1
+==========================================================
+HTTP/1.1 422 Unprocessable Entity
+Content-Length: 88
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "duplicated attribute name and id [attr,]",
```

Comment:
Not really part of this PR, but this extra comma ... would be really nice to have it removed.
New issue?